src/main/java/com/catapult/app/example/services/CallbackServices.java

```diff
@@ -90,160 +150,55 @@ public synchronized void handleOutgoingCallback(final String eventString, final
-                    incomingCallId, userName));
-
-        } catch (Exception e) {
-            LOG.error(MessageFormat.format("Incoming call [{0}] could not be answered", incomingCallId), e);
-        }
+        return bridge;
     }
 
-    // Need to synchronize on instance object to avoid connection allocation problem when calling the api
-    private void bridgeCalls(final Event event, final String userName) {
-        final String incomingCallId = event.getProperty("callId");
-
-        if (userName == null) {
-            LOG.error(MessageFormat.format("Could not find the username for call [{0}]", incomingCallId));
-            return;
-        }
-
-        String outgoingCallId = bridgeMap.get(incomingCallId);
-        if (outgoingCallId == null) {
-            LOG.error(MessageFormat.format("No calls mapped to create bridge for user [{0}] based on call [{1}]",
-                    userName, outgoingCallId));
-            return;
-        }
-
-        Map<String, CallEvents> callEventMap = userEventCallMap.get(userName);
-        if (callEventMap == null) {
-            LOG.error(MessageFormat.format("No call mapped for user [{0}]", userName));
-            return;
-        }
-
-        // Add event when call is answered
-        CallEvents callEvents = callEventMap.get(incomingCallId);
-        callEvents.addEvent(event);
+    private void playRinging(String baseAppUrl, Call call) throws Exception {
+        String url = URLUtil.getStaticResourceUrl(baseAppUrl, "sounds", "ring.mp3");
 
-        try {
-            // Bridge the incoming and outgoing calls using Bandwidth SDK
-            Bridge bridge = Bridge.create(incomingCallId, outgoingCallId);
+        HashMap<String, Object> params = new HashMap<>();
+        params.put("fileUrl", url);
+        params.put("loopEnabled", true);
 
-            if (bridge == null || bridge.getId() == null) {
-                LOG.error(MessageFormat.format("Could not bridge calls [{0}, {1}]",
-                        incomingCallId, outgoingCallId));
-            }
-
-            LOG.info(MessageFormat.format("Calls [{0}, {1}] successfully bridged by [{2}]",
-                    incomingCallId, outgoingCallId, bridge.getId()));
+        call.playAudio(params);
+    }
 
-        } catch (Exception e) {
-            LOG.error(MessageFormat.format("Bridge could not be created for calls [{0}, {1}]",
-                    incomingCallId, outgoingCallId), e);
-        }
+    private Call createCall(final Event event, final String userName, final String to, final String from, final String bridgeId, String baseAppUrl) throws Exception {
+        HashMap<String, Object> params = new HashMap<>();
+        params.put("to", to);
+        params.put("from", from);
+        params.put("tag", event.getProperty("callId"));
+        params.put("callbackUrl", URLUtil.getOutgoingCallbackUrl(baseAppUrl, userName));
+        if (bridgeId != null) params.put("bridgeId", bridgeId);
```

Comment:
Is there a coding convention to always have braces for if statements and on separate lines.